src/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,90 @@
+import { type FC, useState } from "react";
+import { Button, Stack, Typography } from "@mui/material";
+import { signOut } from "next-auth/react";
+import { useRouter } from "next/router";
+
+import { text } from "~/assets/text/index.js";
+import { styles } from "./styles.js";
+import {
+  BotBustersIcon,
+  SoundOffIcon,
+  SoundOnIcon,
+  UserIcon,
+} from "~/assets/icons";
+import { MenuButton } from "~/components/main-menu/menu-button.jsx";
+import { MainMenu } from "../main-menu/index.js";
+import { pages } from "~/router.js";
+
+interface Props {
+  isVerifiedUser: boolean;
+  isGamePlayed: boolean;
+  username: string;
+  open: boolean;
+  setOpen: (open: boolean) => void;
+  disconnect: () => Promise<void>;
```

Comment:
```suggestion
  disconnectWallet: () => Promise<void>;
```

Reply:
implemented the change, watch out when using the suggest feature, if I were to commit that without adjusting other files it would break the app